charts/node/templates/statefulset.yaml

```diff
@@ -635,6 +635,10 @@ spec:
                 --listen-addr=/ip4/0.0.0.0/tcp/${RELAY_CHAIN_P2P_PORT_WS}/ws \
                 {{- end }}
                 {{- end }}
+                --listen-addr=/ip4/0.0.0.0/tcp/30333 \
```

Comment:
Isn't this line should be already covered by https://github.com/paritytech/helm-charts/blob/main/charts/node/templates/statefulset.yaml#L633?

Reply:
RELAY_CHAIN_P2P_PORT is a  `NodePort`  it is dynamic and we have service:
```
NodePort -> 30333
```